src/main/java/io/anserini/index/IndexReaderUtils.java

```diff
@@ -242,6 +242,22 @@ public static IndexReader getReader(String path) throws IOException {
     return termInfo;
   }
 
+  /**
+   * Returns the document frequency of a term. Simply dispatches to <code>docFreq</code> but wraps the exception so
+   * that the caller doesn't need to deal with it; this is potentially dangerous but makes code less verbose.
+   *
+   * @param reader index reader
+   * @param term term
+   * @return tthe document frequency of a term
+   */
+  public static long getDF(IndexReader reader, String term) {
```

Comment:
The function above already provides this functionality right.

Reply:
Correct, but this is a more efficient implementation., doesn't generate intermediate data... which makes a difference if `getDF` is used in an inner loop of some algorithm.